onnxruntime/core/flatbuffers/ort_flatbuffers_py/experimental/fbs/Attribute.py

```diff
@@ -0,0 +1,234 @@
+# automatically generated by the FlatBuffers compiler, do not modify
```

Comment:
>automatically generated by the FlatBuffers compiler [](start = 2, length = 51)

looks like a few generated files are being checked in. is it feasible to generate them on demand (e.g., invoke flatc at build or project generation time)?

Reply:
Flatbuffers is not like Protobuf which has version requirements, so adding generated source codes are easier for now instead of setting up flatc for mobile cross compiling (such as the host_protoc for protobuf), but this can be a future work item 

---
In reply to: [564894823](https://github.com/microsoft/onnxruntime/pull/6428#discussion_r564894823) [](ancestors = 564894823)

Reply:
As these files only need updating when we update the schema, it seemed better to check them in as that also allows someone to run the python scripts to process models without having to build ORT (to trigger flatc being built). 

FWIW we also check in the generated C++ header.